src/components/loggedOut/loggedOutWelcome.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { Typography, Grid, Button } from "@mui/material";
+
+export default function welcomeMessage() {
+  //contains the welcome message for the header to the left
+  return (
+    <Grid
+      item
+      container
+      xs={5}
+      direction="column"
+      p={5}
+      rowGap={2}
+      alignItems="center"
+      color="#F2E8DE"
+    >
+      <Typography variant="h2" textAlign={"center"}>
+        Find Your Perfect Mentor or Mentee with Mesh
+      </Typography>
+      <Typography variant="body1" fontWeight={250} textAlign={"center"}>
+        Unlock Your Potential with Expert Guidance and Support.
+      </Typography>
+      <Button
+        variant="contained"
+        sx={{
+          background: "#68D391",
+          "&:hover": {
+            backgroundColor: "#0A6B57",
+          },
```

Comment:
the hover for both of the buttons is a bit too dark imo, maybe make them a bit more light since the button is already pretty light (kinda like a backdrop effect but for the button)